subprojects/language-groovy/src/integTest/groovy/org/gradle/groovy/GroovyDocIntegrationTest.groovy

```diff
@@ -127,7 +122,9 @@ class GroovyDocIntegrationTest extends MultiVersionIntegrationSpec {
 
         then:
         succeeds 'groovydoc'
-        outputContains '[ant:groovydoc]'
+        if (versionNumber.major < 3) {
+            outputContains '[ant:groovydoc]'
```

Comment:
Why only `< 3`? What happens with 3 with groovydoc?

Reply:
I had a closer look at this test and I think that the test itself is valid for some specific older versions of Groovy (< 2.4.15).
For example, with `1.6.9`, the output matching the pattern is:
```
[ant:groovydoc] line 8:87: unexpected token: >
[ant:groovydoc] ignored due to RecognitionException: pkg/Thing.java [expecting EOF, found '}']
```
Which I think is what we expect as per the code comment above.

Whereas with other versions, the output is
```
[ant:groovydoc] Resource org/apache/groovy/docgenerator/groovy.ico not found so skipped
```
which is present in all the other groovydoc tests with that version and is unrelated to what the test tries to produce.

With Groovy 3 (after the current change) this output (resource not found) goes away.